packages/tabs/src/Tabs/Tabs.tsx

```diff
@@ -48,11 +49,11 @@ import { AccessibleTabsProps } from './Tabs.types';
  * @param props.selected Index of the Tab that should appear active. If value passed, component will be controlled by consumer.
  * @param props.setSelected Callback to be executed when Tab is selected. Receives index of activated Tab as the first argument.
  */
-const Tabs = (props: AccessibleTabsProps) => {
+const Tabs = Polymorphic<AccessibleTabsProps>(props => {
```

Comment:
is `Tabs` itself polymorphic? IIRC the wrapper should always be the same type of element, and only the `TabTitle` can be different

Reply:
updated!